src/duplo_resource/service.py

```diff
@@ -258,6 +257,52 @@ def update_env(self,
       service["Replicaset"] = self.current_replicaset(name)
       self.wait(service, payload)
     return {"message": f"Successfully updated environment variables for service '{name}'"}
+
+  @Command()
+  def update_pod_label(self,
+                 name: args.NAME,
+                 setvar: args.SETVAR,
+                 strategy: args.STRATEGY,
+                 deletevar: args.DELETEVAR,
+                 wait: args.WAIT = False):    
+  
+    """Update the pod labels of a service. If service has no pod labels set, use -strat replace to set new values.
+    Usage: Basic CLI Use
+      ```sh
+      duploctl service update_pod_labels <service-name> --setvar env-key1 env-val1 --setvar env-key2 env-val2 --setvar env-key3 env-val3 -strat merge --host $DUPLO_HOST --tenant $DUPLO_TENANT --token $DUPLO_TOKEN
+      ```
+    Args:
+      name: The name of the service to update.
+      setvar/-V: A list of key value pairs to set as environment variables.
+      strategy/strat: The merge strategy to use for env vars. Valid options are "merge" or "replace".  Default is merge.
+      deletevar/-D: A list of keys to delete from the environment variables.
+    """
+    service = self.find(name)
+    
+    currentDockerconfig = loads(service["Template"]["OtherDockerConfig"])
+    currentLabels = currentDockerconfig.get("PodLabels", [])
+    newLabels = []
+    if setvar is not None:
+      newLabels = [{"Name": i[0], "Value": i[1]} for i in setvar]
+    if strategy == 'merge':
+      d = {d['Name']: d for d in currentLabels + newLabels}
+      mergedlabels = list(d.values())
+      currentDockerconfig['PodLabels'] = mergedlabels
+    else:
+      currentDockerconfig['PodLabels'] = newLabels
```

Comment:
This is for sure wrong. Why are you adding labels as an array? These are objects like so: 
```yaml
labels: 
  my.cool.label: Hello world
```